clusters/ztp-siteconfig/skidoo/skidoo-siteconfig.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "skidoo"
+  namespace: "skidoo"
+spec:
+  baseDomain: "cars.lab"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "img4.9.37-x86-64-appsub"
+# yamllint disable-line rule:line-length
+  sshPublicKey: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAID34lV7EwBk4GDNpnGec0mQdXV3VDLwLiBnvbaikdgLE crucible@oc-client.infra.cars.lab"
```

Comment:
Can we switch to the vse sshKey here?

Reply:
This is mostly for example in this branch and might be removed. Do you think we should still change it?

Reply:
Yes, would like to seed the same SSH key everywhere, and skidoo is an active cluster being used.  I will seed as a part of a PR to add IPv6.